src/DynamoCoreWpf/ViewModels/Core/NodeViewModel.cs

```diff
@@ -1127,8 +1127,13 @@ private void DisposeErrorBubble()
 
         public void UpdateBubbleContent()
         {
+            if (DynamoViewModel == null) return;
+
+            bool hasErrorOrWarning = NodeModel.IsInErrorState || NodeModel.State == ElementState.Warning || NodeModel.State == ElementState.PersistentWarning;
+            if (!NodeModel.WasInvolvedInExecution || !hasErrorOrWarning) return;
```

Comment:
I changed the condition here a bit
Seems to me like if the node WasInvolvedInExecution then there is no need to try and create a bubble

Reply:
unless there can be warning bubbles that do not have anything to do with execution
@QilongTang FYI

Reply:
yes, @pinzart90 - there are warning methods that clients can call at time directly - for Example
`Warning()` 
https://github.com/DynamoDS/Dynamo/blob/master/src/DynamoCore/Graph/Nodes/NodeModel.cs#L1714

Reply:
I'll just put it back as it was...

Reply:
I think you mean node was not involved in execution then do not create a bubble? Yes, this is the current state. However, uur team have a design ready to inject other information regardless if the node is involved in execution or not

Reply:
here is another example:
https://github.com/DynamoDS/Dynamo/blob/80e2bb6b738f3ec7beea699d772251b2faeed708/src/DynamoCore/Models/DynamoModel.cs#L1519